composer.json

```diff
@@ -45,7 +44,7 @@
         "mockery/mockery": "^1.0",
         "phpstan/phpstan-strict-rules": "^0.12",
         "phpunit/phpunit": "^8.0|^9.0",
-        "rector/rector-prefixed": "^0.9.19",
+        "rector/rector": "^0.10.6",
```

Comment:
@50bhan Any reason to use the non prefixed version ? It add lots of dependencies, that's why I used the prefixed version :wink: (non blocker anyway)

Reply:
@Jibbarth Yes, because there were some errors in Rector. I think the "rector/rector" is more reliable and no errors at the moment 😄